.github/workflows/deliver.yml

```diff
@@ -221,9 +321,7 @@ jobs:
     if: github.repository == 'farmOS/farmOS' && github.event_name == 'push' && (github.ref_type == 'tag' || (github.ref_type == 'branch' && needs.build.outputs.farmos_version == '3.x'))
     runs-on: ubuntu-latest
     needs:
-      - build
-      - sniff
-      - test
+      - release
```

Comment:
I think we want to keep this as it was... this is responsible for pushing the `3.x` images to Docker Hub, so we don't want it to only run on `release` (which is for tagged releases).

Reply:
Good catch, that is correct